package.json

```diff
@@ -33,6 +33,7 @@
     "xmlhttprequest": "1.8.0"
   },
   "devDependencies": {
+    "@parse/minami": "^1.0.0",
```

Comment:
Ok I was wondering where this came from, I didn't see  the fork.

Should this build against the master branch?

Reply:
https://github.com/nijikokun/minami/compare/master...parse-community:master

it's published already as an Npm package so we're good.